### ISSUE TITLE
Methods for Adjusting site-specific text

### ISSUE THREAD

**Issue by pbchase** [aandresalvarez/goprod_v2.0#1]
How do you want to handle site-specific text? classes/messages.php is english text, yet it is also specific to Stanford in some places.  These items come to mind:

```
            'INFO_WHAT_NETX_BODY' => '  Your project will be reviewed by SCCI before being moved into Production. This process typically takes 1-2 business days.  Can\'t wait? Click <a href="https://medwiki.stanford.edu/x/SRMzAw"><u>HERE</u></a>.',
            'INFO_CITATION_BODY' => ' REDCap at Stanford is supported by Research IT. All research resulting from the use of REDCap must cite the grants that make this service available - details, including biolerplate language, are located
                <a href="https://medwiki.stanford.edu/x/AAJPB"><u>HERE</u></a>.',
            'MAIN_TEXT' => 'If the thought of losing the data you have entered into your REDCap project sounds painful, you should be in production mode. Production mode helps protect your data from accidental mistakes. This plugin will allow you to verify if your project is ready to move to production mode or if you first need to fix something.  <a href="https://medwiki.stanford.edu/x/SRMzAw" > <u>When do I move to Production Mode?</u></a>',
```

What if you allowed these variables to be configured to change them from the default? That way sites wouldn't have to develop their own language files.

**Comment by aandresalvarez** [comment]
I agree that is something that can help.  I will add it. thanks for the input.

All the best!

**Comment by aldefouw** [comment]
@pbchase  - This is an old request, but I made a provision for this request in my fork of this repository.

https://github.com/aldefouw/goprod_v2.0

My strategy was to make this configurable within the configure section on the "External Modules" configuration page.  (If you click "Configure" button for "Go Prod" module on /ExternalModules/manager/control_center.php)

I also made some adjustments in my fork so that you could configure some thresholds and values to the tsconfig.json file:
![Screen Shot 2021-03-16 at 4 56 31 PM](https://user-images.githubusercontent.com/5683133/111385293-904b8d00-8678-11eb-8745-b8459a83bc05.png)

I just started playing this afternoon with this so there's much more to do, but what a great External Module!  Really think this will help us a ton with these move to production requests.
